Assets/Mirror/Runtime/NetworkWriter.cs

```diff
@@ -19,28 +19,50 @@ public class NetworkWriter
 
         // 'int' is the best type for .Position. 'short' is too small if we send >32kb which would result in negative .Position
         // -> converting long to int is fine until 2GB of data (MAX_INT), so we don't have to worry about overflows here
-        public int Position;
-
+        int position;
         int length;
```

Comment:
Add a getter so that I can get the current length

```suggestion
        int length;
        
        public int Length => length;
```